quai/p2p_backend.go

```diff
@@ -138,6 +138,7 @@ func (qbe *QuaiBackend) OnNewBroadcast(sourcePeer p2p.PeerID, Id string, topic s
 			log.Global.Error("no backend found")
 			return false
 		}
+		txIngressCounter.Add(float64(len(data.WorkObject.Transactions())))
```

Comment:
can you move this to after all the validation of the workshare so that we dont crash